src/main/java/org/fogbowcloud/manager/core/models/orders/Order.java

```diff
@@ -57,6 +59,10 @@ public Token getFederationToken() {
 		return federationToken;
 	}
 
+	public void setFederationToken(Token federationToken) {
+		this.federationToken = federationToken;
+	}
+
```

Comment:
do we really need this setMethod. Can't we pass the arg in the constructor?

Reply:
Spring Boot uses setters and empty constructors to construct objects. This is needed in Order when we receive a create compute request with orders params.

Reply:
Spring Boot uses setters and empty constructors to construct objects. This is needed in Order when we receive a create compute request with orders params.